app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,20 @@
+class CommentsController < ApplicationController
+    def create
+        @post = Post.find(params[:post_id])
+        @comment = @post.comments.new(comment_params)
+        @comment.author = current_user
+
+        if @comment.save
+            flash[:notice] = 'Comment is Ceated!'
+        else
+            flash[:alert] = 'Comment is not Created!'
+        end
+        redirect_to @post
+    end
+
+    private
+
+    def comment_params
+        params.require(:comment).permit(:text)
+    end
+end
```

Comment:
- [ ] Well done working on the add forms however the comments controller should have new and create actions . Kindly add new action.

Reply:
Hello @algerina, Regarding comments, we don't need to add a new action in the controller. Creating a comment on any post always happens on the existing page. If I were to add a new action for comments, it would require navigating to a different page to create a comment on a post, which is not ideal. Please take a look at this